content/articles/momentum-based-gradient-optimizer-and-optimization-techniques-for-gradient-descent/index.md

```diff
@@ -0,0 +1,133 @@
+
+### Momentum-based Optimization
+
+Utilize an Adaptive Optimization Algorithm (AOA) that uses exponentially-weighted averaging of gradients from prior rounds to keep convergence from deviating. Most real-world applications, such as deep neural networks, rely on noisy input to train. 
+
+There must be a systematic approach to feeding the data to minimize the impact of noise during optimization. To resolve this issue, exponentially weighted averages might be used.
+
+Using an iterative process, we can approximate the trends in a P-dimensional noisy dataset by calculating the parameters as we go through the data:
+```
+On iteration p:
+    Get next  θp 
+    v_θ = β vθ + (1 - β)θp
+```
+**Vθ** is averaged across its preceding **1/(1 - β)** iterations, giving a value of **vθ**. With the help of averages, you can eliminate all the randomness and merely keep track of the overall trend. As a result of this method, the algorithm is more resistant to sample noise and can be trained more quickly.
+### How does momentum help in gradient descent
+To describe a gradient descent algorithm, the term `momentum` is used. This method works by detecting the direction of the steepest slope in its present state and then travelling in the direction of that sharpest slope to update its status.
+
+Momentum helps distinguish recent derivatives when Gradient Descent approaches the middle phase of the ravine, and so boosts the gradient descent's direction.
+
+Momentum is a prominent strategy used in conjunction with SGD. When searching, momentum collects the gradients of previous steps and the current ones to decide the best course of action. The following changes have been made to the gradient descent equations.
+
+###  Stochastic Gradient Descent 
+
+Using Stochastic Graduated Descent instead of Classic Gradient Descent, several of the flaws in the classic method are corrected By computing the derivative for each point on the trajectory it follows. It attempts to make up for its lack of computational rigour. It has more considerable noise than Gradient Descent. Hence it takes longer iterations to get to a minimal solution using SGD. In contrast to Gradient Descent, SGD computes the derivatives for all points at the same time.
+### Adaptive Optimization Algorithm techniques.
+In the next section, we'll take a look at a few approaches used in AOAs.
+### Adaptive Moment Estimation
+RMSProp and momentum combine to create this effect. Use this method to figure out how fast your brain is adapting to new information. Like momentum, it saves the declining arithmetic mean of the squared gradients and an average of the past gradients, which is a similar feature.
+
+On a smooth surface, it's considerably easier for Adam to make a mistake because of his high coefficient of friction. So, for example, this strategy's pseudocode appears like this:
+```
+D = 0

```

Comment:
use latex

Reply:
This is not latex. Latex equations begin with $ and end with $. Please check published articles with equations in them and update accordingly.